### ISSUE TITLE
Can have more details on the ServerGameSettings?

### ISSUE THREAD

**Issue by forleong** [StunlockStudios/vrising-dedicated-server-instructions#8]
Like DeathContainerPermission what value can we set on this? Better have a document list out all the available option of it, Thanks!

**Comment by HackSassin** [comment]
I would also very much like to have the the potential variables and what they would do. I would also like this document. Thank you :)

**Comment by raykai** [comment]
same here would like more information

**Comment by JustPortuguese** [comment]
I would love a small explanation of each parameter and the range of values on them as well!

Would help so much. Thank you for the game! Been having so much fun.

**Comment by leonard4** [comment]
Anyone know what the option for disabling castle heart destruction it? Tried changing CastleHeartDamageMode to "Never", but then my server doesn't load.

**Comment by vachiko** [comment]
I also hope the author can provide detailed information.

**Comment by Shags3379** [comment]
https://cdn.stunlock.com/blog/2022/05/25083113/Game-Server-Settings.pdf

**Comment by drakmirv** [comment]
> > Anyone know what the option for disabling castle heart destruction is? Tried changing CastleHeartDamageMode to "Never", but then my server doesn't load.
> 
> Direct from our current config file
> 
> ` "CastleDamageMode": "Never",`

I don't think this stops destroying hearts.  In fact, if the castle goes into "decay", there doesn't appear to be a way to stop it from being destroyed with any of the settings.  (And the settings to "delay" the decay as much as possible are not working correctly right now)

_**Seems to be a common request in Discord - to remove the Castle Decay mechanism for dedicated servers.**_
